amundsen_application/static/js/ducks/search/api/v0.ts

```diff
@@ -27,11 +27,18 @@ export const searchResourceHelper = (response: AxiosResponse<SearchAPI>) => {
   return ret;
 };
 
-export function searchResource(pageIndex: number, resource: ResourceType, term: string) {
+export function searchResource(pageIndex: number, resource: ResourceType, term: string, filters?: {}) {
   if (resource === ResourceType.dashboard ||
      (resource === ResourceType.user && !AppConfig.indexUsers.enabled)) {
     return Promise.resolve({});
   }
+  if (filters) {
+    return axios.post(`${BASE_URL}/${resource}_qs`, {
+      filters,
+      term,
+      pageIndex
+    }).then(searchResourceHelper);
+  }
```

Comment:
This might still end up being two APIs because one is a `get` while the other is a `post`. Explorations on the frontend regarding the behavior of the current search endpoint will affect what gets done here